.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ 2.5, 2.6, 2.7, 3.0, jruby-9.3 ]
+        ruby: [ 2.5, 2.6, 2.7, '3.0', 3.1, jruby-9.3 ]
```

Comment:
Why do we need to put 3.0 in single quotes ?

Reply:
See the comment and screenshot above in the PR.  An unquoted 3.0 is truncated to 3.

Reply:
Ah my bad, didn't read the details. I thought we are only adding 3.1